proposals/0003-pr-process.md

```diff
@@ -0,0 +1,22 @@
+---
+author: Tikhon Jelvis
+date-accepted: N/A
+discussion: https://github.com/haskell-org/committee/pull/3
+---
+
+# Accepting Website PRs
+
+To quickly triage and accept PRs to the website, we need a process for reviewing, merging and deploying changes to the website. We adopted a process for this before we had this proposal repo, and the process helped us respond to and merge PRs faster than before. This proposal codifies that process to help make our policies explicit and transparent.
+
+Guidelines for accepting PRs to the website:
+
+  1. Purely technical fixes—like fixing the `default.nix` file, broken links, typos and HTML glitches—require one member of the committee to approve.
+  2. Uncontroversial changes that affect the content of the site need two committee members to approve. If the PR was authored by one or more committee members, it should be approved by *other* members unless there are less than two members not involved in the PR.
```

Comment:
I'm sceptical of the value of requiring non-author members to approve. Firstly if two committee members co-authored an uncontroversial PR then we already have reason to believe it will be a good PR. Secondly this rule is very easy to subvert by a committee member asking a conspirator non-committee member to submit the PR on their behalf. We don't want to cause so much frustration that subverting the rule becomes a tempting course of action!

Reply:
And to add a bit of colour, suppose two committee members collaborate on producing a nice update to the website. There are only five other committee members. The policy as stated requires two of these five busy people to find time to review the PR. Reviews are *already* hard enough to come by. This particular policy would make the situation worse.

Reply:
Yeah. I've worked on a few PRs since I first wrote up this policy, and it definitely feels like it's both unnecessary *and* adds enough friction to slow PRs down substantially.

Reply:
I suggest changing this to

```
2. Uncontroversial changes that affect the content of the site need two committee members to approve. If the PR was authored by one or more committee members those committee members are still allowed to be approvers.
```